thangs_importer.py

```diff
@@ -282,50 +283,14 @@ def import_model(self):
             print("File Path:", self.file_path)
             print("File Extension:", self.file_extension)
 
+            ImportObject = None
```

Comment:
Python conventions are for local variables to be `snake_case`.  So, I suggest `import_result` or something like that as the name of this variable.

So, with the other re-namings this line would be more like
```
import_result = import_model(self.file_extension, self.file_path)
```